### DISCUSSION TITLE
Filtering on Teams do not work?

### DISCUSSION THREAD

**Discussion by kjetildk** [community/community#39132] [Code Security]
### Select Topic Area

Bug

### Body

Hi,

In the new Risk and Coverage dashboards the filter for teams do not work. The list is empty when selecting a team.

Is this a bug? Or do we need to make any global changes to our Enterprise to make this work?
Please advise :-)

Regards
Kjetil

**Comment by kjetildk** [comment]
The team need to have Admin role access to the repo to make it visible in the dashboard. Is this as planned?

Shouldn't the "Developers (Write+Security Overview)" role also work?

**Reply by kellyarwine** [reply]
Thanks for the feedback @kjetildk.  We hear you.  Would you be open to talking with the security overview product team about what would work for you here?  If so, here's a link to [book time with me](deleted).  As a thank you, I can send you a $75 gift card from your choice of retailers. (I will remove this link in the next 48 hours to avoid spam. 😄 )
